### PR TITLE
GUI enhancement: Find in Finder dlg

### LINKED ISSUE #13624: [GUI] Find in Finder key accelerator
### Description of the Issue

image
Missing key accelerator.
### Steps to Reproduce the Issue

1. Open Find dialog (ctrl+f)
2. Press Find all in Current Document button -> Search result panel will open
3. Right-click in panel and left-click on Find in these search results...
4. Press ALT key to enable key accelerators

### Expected Behavior

Key accelerator.
### Actual Behavior

No key accelerator.
### Debug Information

Notepad++ v8.5.2

### PR DESCRIPTION
**Pull request by ozone10**
add key accelerator
optimize dark mode
add initializers
fix override warnings
image

fix #13624

### CODE REVIEW

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc [hunk 1]
```diff
@@ -115,28 +115,28 @@ EXSTYLE WS_EX_TOOLWINDOW | WS_EX_WINDOWEDGE
 CAPTION "Search results"
 FONT 8, TEXT("MS Shell Dlg"), 0, 0, 0x0
 BEGIN
-DEFPUSHBUTTON   ">",IDC_INCFINDNXTOK,243,0,16,14, NOT WS_VISIBLE
+//DEFPUSHBUTTON   ">",IDC_INCFINDNXTOK,243,0,16,14, NOT WS_VISIBLE
```
Comment (Yaron10) [inline PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc:118]: @ozone10,

Thank you for the "Dialogs Improvements" series. Great work! 👍 

It seems that commenting this line creates an issue:
_Find in Current Document_.
Again, _Find in Current Document_.

_Result_:
The entire text in the _Search Results_ window is selected.

Comment (ozone10) [reply PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc]: Good catch, I've added comment

**Comment by ozone10** [comment]
Fix merge conflict

**Comment by donho** [comment]
@ozone10 
It's much prettier now.
But could you make these 3 controls a little bit wider please?

image

**Review by donho — CHANGES_REQUESTED** [review]
make 3 controls wider for the sake of translation.

**Review by donho — APPROVED** [review]

**Pull request closed without merging**